website/docs/guides/PAK-upgrade-guide-1.10.0.html.markdown

```diff
@@ -9,6 +9,8 @@ MongoDB Atlas Provider : Upgrade and Information Guide
 # MongoDB Atlas Provider: PAK Upgrade Guide in v1.10.0
 In Terraform MongoDB Atlas Provider v1.10.0, some improvements were introduced which mainly focus on the MongoDB Atlas Programmatic API Keys (PAK) handling. This guide will help you to transition smoothly from the previous version which this resource was first released (v1.8.0) to the new version (v1.10.0).
 
+For comprehensive Upgrade Guide on all v1.10.0 modifications see [here](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/1.10.0-upgrade-guide). 
```

Comment:
Highly recommend spelling out PAK in the title and only using PAK in the text after it is introduced in line 10.

Reply:
updated all headers accordingly here too